src/components/ebay-combobox/template.marko

```diff
@@ -26,7 +26,7 @@
             aria-haspopup='listbox'
             w-on-blur='handleComboboxBlur'
             w-on-keyup='handleComboboxKeyUp'
-            autocomplete="off"
+            autocomplete="new-password"
```

Comment:
What does this "new-password" value mean/do? Needs a description in the PR and a comment in the code.

Reply:
Actually, this may not be working as expected anyway. Let me confirm it even works as expected. I'll close the PR for now.